CONTRIBUTING.md

```diff
@@ -0,0 +1,161 @@
+
+The following fields are allowed as of right now:
+
+- name
+- bio _(optional)_
+- url _(optional)_
+- twitter _(optional)_
+- github _(optional)_
+
+## Step 3: Adding your new post to the O-Devs blog
+
+Now that your username is under the list of authors, we get to the fun part: adding  your article to our blog.
+
+To get a grasp on we're about to do here, we highly recommend you the [Jekyll docs about Posts](http://jekyllrb.com/docs/posts).
+
+**1. Add a new file to `_posts/` with the `.md` extension following these instructions**
+
+- Use this convention when naming your file: `YEAR-MONTH-DAY-title.MARKUP`
+
+- Here's an example: `_posts/2015-11-30-i-love-the-odevs.md`
+
+**2. Add content to your newly created file**
+
+Here is an example:
+
+```
+
+---
+layout: post
+title:  "Why you should use Angular"
+date:   2015-11-12 14:30:39
+categories: angular javascript
+author: sergiocruz
+---
+You should use Angular because it's trendy. Here's how to create a directive:
+
+{% highlight javascript %}
+angular
+  .module('app')
+  .directive('foo', function() {
+    return {
+      restrict: 'E',
+      template: 'Sup yo'
+    }
+  });
+{% endhighlight %}
+```
+
+We'll separate this file in two parts: **header** & **body**.
+
+### Header
+
+This is the information that lives in the top of each post, between the dashes (`---`).
+
+Be sure to change the information on the header of your post file, it is very important that you add the following fields on the files on the header:
+
+- layout _(should always be "post")_
+- title: _(your post's title)_
+- date: _(when your article was written)_
+- categories: _(category tags separated by spaces)_
+- author: _(same username used on `_data/authors.yml`, same as your github account)_
+
+### Body
+
+The body is where the actual content of your article will be located. Be sure to be comfortable using markdown to write content.
+
+In this blog we use the `kramdown` flavor of markdown, but it's pretty similar to github's flavor of markdown.
+
+**3. Previewing your post**
+
+You'll need to install Jekyll on your machine to preview your blog post. Be sure to preview your post before making a pull request. This is a community-based blog and we expect you to bullet-proof your article before making a pull request. By bullet-proofing we mean making sure that your content is properly formatted, etc.
+
+- Learn how to [install Jekyll](http://jekyllrb.com/docs/installation/)
+
+- Serve Jekyll locally by running this command on your terminal: `jekyll serve`
+
+The output of `jekyll serve` should look something like this:
+
+```
+> jekyll serve
+Configuration file: orlandodevs.github.io/_config.yml
+            Source: orlandodevs.github.io
+       Destination: orlandodevs.github.io/_site
+      Generating... done.
+ Auto-regeneration: enabled for 'orlandodevs.github.io'
+Configuration file: orlandodevs.github.io/_config.yml
+    Server address: http://0.0.0.0:4000/
+```
```

Comment:
This whole section should provide the instructions for the Github Pages setup. They'll need Ruby installed. Then they can just `bundle install` and `bundle exec jekyll serve` to make sure the stuff they're doing always works with Github's Jekyll installation.